### ISSUE TITLE
Optimize Lighthouse Chrome Score

### ISSUE THREAD

**Issue by Manuel-Suarez-Abascal** [SidKwok/github-explorer#1]
I would like to work on increasing Lighthouse Chrome Score on your project. Let me know if I get green light to work on this issue. You can see the current score in the image below:

![Lighthouse_chrome_score](https://user-images.githubusercontent.com/32891808/59566199-a010ea00-902a-11e9-93ec-efc42531fb61.jpg)

**Comment by SidKwok** [comment]
It would be great!

**Comment by Manuel-Suarez-Abascal** [comment]
@SidKwok awesome, I will work it tonight!

**Comment by Manuel-Suarez-Abascal** [comment]
@SidKwok I have fixed lighthouse score. I wanted to assign the merge request to you, so you were able to review my code, but I think I made a mistake and merge before letting you review it on master. Feel free to revert the changes on master, sorry about that. 

**This is the current score:**

![lightscore-score-fixed](https://user-images.githubusercontent.com/32891808/59571046-901af980-906e-11e9-9fa8-cd2e5a0e7290.jpg)
